tests/Databases.tests.ps1

```diff
@@ -23,6 +23,14 @@ Describe "SQL Server Databases" {
                 (Get-DatabasesWithoutDDLTrigger -ServerInstance $serverInstance -TriggerName $MustHaveDDLTrigger).Count | Should Be 0
             }
 
+            It "$serverInstance has all databases under Max DataFile Space Used" {
+                $MaxDataFileSpaceUsedPercent = $config.MaxDataFileSpaceUsedPercent
+                if($MaxDataFileSpaceUsedPercent  -eq $null) {
+                    Set-TestInconclusive -Message "No config value found"
+                }
+                (Get-DatabasesOverMaxDataFileSpaceUsed -ServerInstance $serverInstance -MaxDataFileSpaceUsedPercent $MaxDataFileSpaceUsedPercent).Count | Should Be 0
```

Comment:
Would be good to add an example to the config file to make it very clear how the percent should be specified (e.g. 90 not 0.9).